src/app/templates/test/_plugin.test.js

```diff
@@ -0,0 +1,54 @@
+import document from 'global/document';
+
+import QUnit from 'qunit';
+import sinon from 'sinon';
+import videojs from 'video.js';
+
+import plugin from '../src/plugin';
+
+const Player = videojs.getComponent('Player');
+
+QUnit.test('the environment is sane', function(assert) {
+  assert.strictEqual(typeof Array.isArray, 'function', 'es5 exists');
+  assert.strictEqual(typeof sinon, 'object', 'sinon exists');
+  assert.strictEqual(typeof videojs, 'function', 'videojs exists');
+});
+
+QUnit.module('<%= nameOf.package %>', {
+
+  beforeEach() {
+    this.player = new Player(document.createElement('video'));
```

Comment:
That makes sense. I'll move the element.

We should definitely address the test helpers, but not in this PR. :)